Dockerfile

```diff
@@ -3,14 +3,18 @@
 FROM ubuntu:jammy-20221130
 RUN apt-get update -qq \
            && apt-get install -y -q --no-install-recommends \
-                  ca-certificates curl apt-utils \
+                  ca-certificates curl apt-utils gnupg \
            && rm -rf /var/lib/apt/lists/*
+RUN mkdir -p /etc/apt/keyrings
+RUN curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+ENV NODE_MAJOR="18"
+RUN echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_$NODE_MAJOR.x nodistro main" | tee /etc/apt/sources.list.d/nodesource.list
 RUN curl -sL https://deb.nodesource.com/setup_18.x | bash -
```

Comment:
I think this line should not be there

Reply:
you're right, that was to be removed as proposed here https://github.com/bids-apps/base_validator/pull/11/files

Reply:
https://github.com/bids-apps/base_validator/pull/13